pypeclub/OpenPype
# Maya: Handle excluding `model` family from frame range validator.

## Brief description
This resolves #2562 by excluding the `model` and some other families from having their `frameStart` and `frameEnd` attributes validated (which they wouldn't have because of their being geometry).

## Description
The `validate_frame_range.py` currently validates the incorrect subset for published Review instances when there's a nested `model` subset in it.  This fix bypasses the validator checks when it detects the families are of `model` instance, and some others.


## Testing notes:
1. Create model and then create OP `model` instance.
2. Create camera and then create OP `review` instance.
3. Append model instance subset into review subset.
4. Publish.
5. If frame range is set incorrectly, it should give an error and the validator should be able to repair that error by conforming the review instance frame range to the one set in the mongo document.
